### PR TITLE
Finished project

### PR DESCRIPTION
**Pull request by JamesLizamore**

### CODE REVIEW

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@JamesLizamore You'll need to remove your .git folder from inside your project and re-commit it. It was picked up as a submodule: 

image

File: Calculator.JamesLizamore/Calculator
Comment (JamesLizamore) [inline Calculator.JamesLizamore/Calculator:1]: I assume it wrong again as there is still a submodule?

Comment (TheCSharpAcademy) [reply Calculator.JamesLizamore/Calculator:1]: Yes. Maybe just start again. 
1 - Delete the parent repository
2 - Clone it again
3 - Remove git folder from your project
4 - Copy your project into the cloned repo and create PR

**Comment by TheCSharpAcademy** [comment]
@JamesLizamore Still hasn't worked. If you click on the "files changed" tab you can see what files you submitted for review, as per picture below. Could you please submit your own repository in your github so I can review it? I don't want to slow you down 😊

image

**Comment by JamesLizamore** [comment]
https://github.com/JamesLizamore/Calculator2

**Comment by TheCSharpAcademy** [comment]
@JamesLizamore approved 😁

**Pull request closed without merging**